docs/go/quickstart/users-profile-sample-application.md

```diff
@@ -0,0 +1,179 @@
+---
+id: users-profile-sample-application
+title: Users Profile Sample Application
+sidebar_label: Gin-Mongo Sample 2
+tags:
+  - hello-world
+  - go
+  - sdk
+  - tutorial
+---
+
+# Users-Profile
+
+A sample application that get, create, update, and delete the data of a user in the database (MongoDB for this application).
+
+## Prerequisites
+
+1. [Go](https://go.dev/doc/install) 1.16 or later
+2. [Docker](https://docs.docker.com/engine/install/) for running Keploy server
+3. [Thunder Client](https://marketplace.visualstudio.com/items?itemName=rangav.vscode-thunder-client) / [Postman Desktop Agent](https://www.postman.com/downloads/postman-agent/) for testing localhost APIs
+4. Code Editor ([VSCode](https://code.visualstudio.com/download), [Sublime Text](https://www.sublimetext.com/download), etc.)
+
+## Installation
+
+### Start Keploy Server
+
+
+git clone https://github.com/keploy/keploy.git
+cd keploy
+docker-compose up
+
+
+### Start Users-Profile sample application
+
+
+git clone https://github.com/keploy/samples-go
+cd samples-go
+cd users-profile
+go get .
+go run .
+
+
+## Routes
+
+> Sample Application Port: http://localhost:8080
```

Comment:
This looks like a normal text. Can you use <a> tag here instead?

![Screenshot from 2022-10-04 16-17-30](https://user-images.githubusercontent.com/74777863/193802109-71ab11a8-c9f0-4561-9ec2-746b7a520dff.png)